packages/dev/test/RelayHubGasCalculations.test.ts

```diff
@@ -32,6 +35,12 @@ const TestPaymasterVariableGasLimits = artifacts.require('TestPaymasterVariableG
 const TestPaymasterConfigurableMisbehavior = artifacts.require('TestPaymasterConfigurableMisbehavior')
 const RelayRegistrar = artifacts.require('RelayRegistrar')
 
+const contractOrig = contract
+if (process.env.GAS_CALCULATIONS == null) {
```

Comment:
this is ugly. a much better way is 
```
before(function() { if (...) this.skip() } )
```
and maybe it also work at the "contract" level.

Reply:
In my experience `this.skip()` works inconsistently. If this is important I can try, but I suggest ignoring the ugliness of this, as `if (process.env.GAS_CALCULATIONS == null) this.skip()` isn't much better.
Better way in my opinion would be to keep tests in separate folders, but this shouldn't be a part of this PR.